cmd/manager/collection/main.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"github.com/kabanero-io/kabanero-operator/pkg/apis"
+	"github.com/kabanero-io/kabanero-operator/pkg/controller/collection"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
```

Comment:
When I did the admission webhook I removed all of the `operator-sdk` imports and tried to remove as much of that code as I could so that it was clear that the admission webhook was not generated from `operator-sdk`.  I would advise to do the same here.  I did pass in the namespace as an environment variable so that when I created the controller and the watches, I would know what namespace to use.